mappings/net/minecraft/sortme/LevelIndexedProcessor.mapping

```diff
@@ -0,0 +1,80 @@
+CLASS cep net/minecraft/sortme/LevelIndexedProcessor
```

Comment:
`Level` could possibly become confusing due to being used for world saves.

Reply:
As explained before, the reason I named it like that was that there is evidence in the form of exception message that mojang names them `levels`. I don't really agree with that name, but if you have any better idea - I would like to see it.